sonar-html-plugin/src/test/java/org/sonar/plugins/html/core/HtmlSensorTest.java

```diff
@@ -107,6 +107,21 @@ public void testSensor() throws Exception {
     assertThat(tester.allAnalysisErrors()).isEmpty();
   }
 
+  @Test
+  public void should_not_count_doctype_and_html_tag_in_duplication() throws Exception {
+    DefaultInputFile inputFile = createInputFile(TEST_DIR, "create-salesorder_duplication.xhtml");
+    tester.fileSystem().add(inputFile);
+
+    sensor.execute(tester);
+
+    String componentKey = inputFile.key();
+    // 206 tokens with no filtering
+    assertThat(tester.cpdTokens(componentKey)).hasSize(196);
```

Comment:
Given the limited computing capabilities of my brain, I would have preferred a smaller test case.